src/PhpEnvironment/Request.php

```diff
@@ -95,7 +95,7 @@ public function getContent()
     {
         if (empty($this->content)) {
             $requestBody = file_get_contents('php://input');
-            if (strlen($requestBody) > 0) {
+            if (! empty($requestBody)) {
```

Comment:
`empty()` not needed: `!` is sufficient

Reply:
for `bool` comparison usage, with `if ($requestBody) {` should be enough